src/extract_topic.py

```diff
@@ -1,64 +1,233 @@
-    for index, topic in enumerate(model.components_):
-        message = "\nTopic #{}:".format(index)
-        message += " ".join([feature_names[i] for i in topic.argsort()[:-n_top_words - 1 :-1]])
-        print(message)
-        print("="*70)
-
-
-def print_topic(text_dir, out_dir, n_top_words = 10):
-    text = generate_text_li(text_dir)
-    tf, tf_vectorizer = process_text2vec(text)
-    lda = model(tf, n_comp = 4)
-    #print("\n Assigning Topics in LDA model: ")
-    feature_names = tf_vectorizer.get_feature_names()
-    with open(out_dir+'/topic.txt', 'w') as f:
-        for index, topic in enumerate(lda.components_):
-            f.writelines("\n"+" ".join([feature_names[i] for i in topic.argsort()[:-n_top_words - 1 :-1]]))
-            
- 
+import re
+import nltk
+from pytorch_pretrained_bert import BertTokenizer
+import time
+from bert_serving.client import BertClient
+from bert_serving.server import BertServer
+from bert_serving.server.helper import get_args_parser
+from sklearn.feature_extraction.text import TfidfVectorizer
+
+
+def bert_tokens(text_all, token_choice='bert-base-uncased'):
+    tokenizer = BertTokenizer.from_pretrained('bert-base-uncased')
+    text_article_tokenized = []
+    text_flat_tokenized = []
+    for text in text_all:
+        text_tokenized =[]
+        tmp_tokenized=[]
+        for sentence in text:
+            toks = tokenizer.tokenize(sentence)
+            for tok in toks:
+                tmp_tokenized.append(tok)
+            text_tokenized.append(toks)
+        text_flat_tokenized.append(tmp_tokenized)
+        text_article_tokenized.append(text_tokenized)
+    return text_flat_tokenized, text_article_tokenized
+
+
+def get_topic_embedding(topics, port=6000, port_out=6001, model_path='/home/ubuntu/bert_tests/bert-as-service/uncased_L-12_H-768_A-12'):
+    common = [
+        '-model_dir', model_path,
+        '-num_worker', '2',
+        '-port', str(port),
+        '-port_out', str(port_out),
+        '-max_seq_len', '20',
+        '-max_batch_size', '256',
+        '-pooling_layer', '-2',
+        '-cpu',
+    ]
+    args = get_args_parser().parse_args(common)
+    server = BertServer(args)
+    server.start()
+    print('wait until server is ready...')
+    time.sleep(20)
+    print('encoding...')
+    bc = BertClient(port=port, port_out=port_out, show_server_config=False)
+    vec = bc.encode(topics)
+    bc.close()
+    server.close()
+    np.save('./out/topic_embedding',vec)
+    return vec
+
+
+def tfidf_vec(text_flat_tokenized, stopwords):
+    """returns tfidf weight for each word in each issue"""    
+    def dummy_doc(doc):
+        return doc
+
+    vectorizer = TfidfVectorizer(
+        analyzer='word',
+        stop_words=stopwords,
+        tokenizer=dummy_doc,
+        preprocessor=dummy_doc,
+        token_pattern=None)  
+    X = vectorizer.fit_transform(text_flat_tokenized)
+    feature_names=vectorizer.get_feature_names()
```

Comment:
Add spaces before and after `=`. Similarly, this applies to all other places. Please be consistent.